### PR TITLE
Move code sample to a separate file and add a prerelease banner

### PR DESCRIPTION
**Pull request by MatthewVaadin**
This extracts a code sample to its own file for two reasons:
1. So that everything required to get the code sample working is available on the page.
2. So that the validity of the code is checked and changes to the API will force the example to change.

### CODE REVIEW

File: articles/tools/observability/configuration.asciidoc [hunk 1]
```diff
@@ -99,7 +99,11 @@ otel.instrumentation.${instrumentationName}.enabled=true
...
 
 
-== Frontend Observability Configuration
+:prerelease-feature: Frontend Observability Configuration
+include::{articles}/_prerelease-banner.asciidoc[opts=optional]
+
+Frontend observability is enabled by default, with all the client-side instrumentation active.
+The configuration can be tuned in two ways:
 
 Frontend observability is enabled by default, with all of the client-side instrumentation active. The configuration can be tuned in two ways: statically by editing the [filename]`agent.properties` file, or by providing environment variables or system properties; or dynamically by implementing the [interfacename]`ObservabilityClientConfigurer` interface, which applies changes at runtime.
```
Comment (jouni) [inline articles/tools/observability/configuration.asciidoc:108]: Duplicate content.

File: articles/_prerelease-banner.asciidoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+</style>
+++++
+
+ifndef::prerelease-feature[]
+:prerelease-feature: this feature
+endif::[]
+
+ifndef::prerelease-banner-content[]
+:prerelease-banner-content: The {prerelease-feature} feature has not yet been released and is only available through the prerelease repository.
```
Comment (jouni) [inline articles/_prerelease-banner.asciidoc:62]:
```suggestion
:prerelease-banner-content: The {prerelease-feature} feature has not yet been released and is only available through the pre-release repository.
```

**Review by jouni — COMMENTED** [review]
Is this information enough for the reader to use the pre-release feature? Personally, I might need a bit more hand-holding to know what to do with the pre-release repo link.

**Comment by MatthewVaadin** [comment]
The changes here will be re-applied in a combined PR for V2.1 of Observability Kit.

**Pull request closed without merging**